pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -550,3 +551,18 @@ def test_timestamp_constructor_identity():
     expected = Timestamp("2017-01-01T12")
     result = Timestamp(expected)
     assert result is expected
+
+
+@pytest.mark.parametrize("kwargs", [{}, {"year": 2020}, {"year": 2020, "month": 1}])
+def test_constructor_missing_keyword(kwargs):
+    # GH 31200
+
+    # The exact error message of datetime() depends on its version
```

Comment:
rather than doing this

just construct a regex of the possible messages and ```|``` separate

Reply:
done